lib/bolt_ext/server.rb

```diff
@@ -6,10 +6,17 @@
 require 'json'
 
 class TransportAPI < Sinatra::Base
+  # This disables Sinatra's error page generation
+  set :show_exceptions, false
+
   get '/' do
     200
   end
 
+  get '/500_error' do
```

Comment:
This is specifically for testing?

Reply:
Yeah - I'm not married to the idea of adding it. It wasn't clear if there was a better way to test the 500 error with rspec, and I didn't want to sink more time into it. I'm not especially worried about the 500 error breaking since it's formatted just like the 404 though, so I'd be fine with removing this.

Reply:
I think it's fine.